docs/requirements.txt

```
@@ -0,0 +1,12 @@
+sphinx
+sphinx_rtd_theme
+jupyter
+nbsphinx
+wget
+numba
+numpy
+matplotlib
+scipy
```

For code highlighting in blocks:
```suggestion
pygments
scipy
```